index.html

```diff
@@ -33,23 +33,131 @@ <h1>This is a branch called part 1</h1>
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a class="navbar-brand" href="index.html">
-                    <h1 class="text-center">Chefs Table</h1>
-                    <div class="clearfix"></div>
-                </a>
+              <a class="navbar-brand" href="index.html">
+                <h1 class="navbar-brand">Chefs Table</h1>
+                <div class="clearfix"></div>
+              </a>
             </div> <!-- /.navbar-header -->
 
             <!-- Collect the nav links, forms, and other content for toggling -->
             <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-                <ul class="nav navbar-nav navbar-right">
-                    <li><a href="#about-section">About</a></li>
-                    <li><a href="#menu-section">Explore the Menu</a></li>
-                    <li><a href="#reservation-section">Make a Reservation</a></li>
-                </ul>
+              <ul class="nav navbar-nav navbar-right">
+                <li><a href="#home-section">About</a></li>
+                <li><a href="#about-section">Explore the Menu</a></li>
+                <li><a href="#menu-section">Make a Reservation</a></li>
+              </ul>
             </div><!-- /.navbar-collapse -->
         </div><!-- /.container -->
     </nav><!-- /.navbar -->
 
+		<section class="container">
+			<div class="row">
+				<div class="col-xs-12 col-sm-12 col-md-4 col-md-push-1">
+					<h2>A Unique Culinary Experience</h2>
+					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nesciunt voluptates, voluptatum mollitia. Libero, veritatis ducimus eos quaerat maxime, asperiores laborum quam temporibus, est ratione atque rem voluptatibus ullam dicta non sequi perferendis eaque consequuntur qui soluta, distinctio quisquam excepturi nisi fugiat culpa! Nulla pariatur, accusantium id at nihil cumque ut.</p>
+				</div>
+				<div class="col-xs-12 col-xs-push-1 col-sm-12 col-sm-push-2 col-md-4 col-md-push-1">
+					<img src="img/blueberries.jpg" alt="parfait" class="image-resize">
```

Comment:
class="image-resize"

use bootstrap class [img-responsive](https://getbootstrap.com/docs/3.3/css/#images) to handle resizing this image
class="img-responsive"